### DISCUSSION TITLE
Ubuntu Desktop on Orange Pi 5 Pro

### DISCUSSION THREAD

**Discussion by cantux-fr** [Joshua-Riek/ubuntu-rockchip#781] [Q&A]
On an Orange Pi 5 Pro, I installed 'ubuntu-24.04-preinstalled-desktop-arm64-orangepi-5-pro.img.xz' using Balena Etcher onto a micro SD card. Ubuntu does not start. If I do the same thing with the official OrangePi version, it works. Can you tell me how to fix this? Otherwise, thank you very much to the developers who worked on this.

**Comment by cantux-fr** [comment]
It's works with Ubuntu 22.

**Comment by kmn1024** [comment]
I think many people would be interested to learn about what you did: https://github.com/Joshua-Riek/ubuntu-rockchip/issues/768. Could you help the community out by describing what you did, and which components work? (e.g. did you get WiFi to work?)

**Comment by cantux-fr** [comment]
I downloaded version 22.04 instead of 24.04 from this address: https://github.com/Joshua-Riek/ubuntu-rockchip/discussions/781. The 22.04 works well, but the WiFi does not work (networks not detected) and Bluetooth does not work either.
